### PR TITLE
DEVOPS-2908 adding healthz endpoint

### PR DESCRIPTION
**Pull request by dskatz**

### CODE REVIEW

**Review by whwalter — APPROVED** [review]

**Review by yuzhouliu9 — APPROVED** [review]
LGTM

**Comment by whwalter** [comment]
@dskatz would adding this as boilerplate to https://github.com/10gen/kanopy-app-gohttps://github.com/10gen/kanopy-app-go make sense?

**Comment by dskatz** [comment]
> @dskatz would adding this as boilerplate to https://github.com/10gen/kanopy-app-gohttps://github.com/10gen/kanopy-app-go make sense?

I think that would make sense. I'll add a ticket for that.

File: internal/server/server.go [hunk 1]
```diff
@@ -117,6 +120,21 @@ func (s *Server) handleRoot() http.HandlerFunc {
...
 
+func (s *Server) handleHealthz() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		status := map[string]string{
+			"status": "ok",
+		}
+
+		bytes, err := json.Marshal(status)
+		if err != nil {
+			logAndError(w, http.StatusBadRequest, err, "error gathering status")
```
Comment (colinhoglund) [inline internal/server/server.go:131]: @dskatz You need a `return` after this error.

Comment (dskatz) [reply internal/server/server.go:131]: doh!

**Pull request merged**